README.md

```diff
@@ -1,28 +1,143 @@
 # VirtoCommerce.Marketing
+
 VirtoCommerce.Marketing module represents a marketing management system.
-Key features:
-* promotions 
-* dynamic content management for storefront
-* content publishing management
 
-![Marketing UI](https://cloud.githubusercontent.com/assets/5801549/15642380/9767eaae-264f-11e6-8d63-e2d53129646d.png)
+**Key features**:
+
+* Dynamic content management for storefront
+* Content publishing management
+* Promotions
+
+## Dynamic Content
+
+The Dynamic Content feature allows marketers to create personalized shopping experiences for customers. Using this feature, the marketer can deliver elements of a shopping experience based on a set of conditions or context.
+
+The following should be considered before determining under which conditions the Dynamic Content will be displayed to the customer:
+
+* What type of content should be displayed;
+* The target audience who will see the content;
+* Time period when the content should be displayed;
+* Where should the content be displayed.
+
+The Dynamic Content is configured, using the Virto Commerce platform manager so that the marketing personnel can set up Dynamic Content without relying on developers and server administrators. For images and Flash animations, they may need a graphic designer to create the file and upload it to the Assets. However, once that's done, the marketer can configure rules for displaying the content without any assistance from the technical team side.
```

Comment:
Are you sure? Do I have to remove this?

Reply:
The Dynamic Content is configured, using the Virto Commerce platform manager

IMHO , is not needed for sure here. 
As I understand it is one straight phrase - telling that the Dynamic Content could be configured using the Virto Commerce platform manager. I do not see clarification here, which could be quoted with commas. In Russian there should not be a comma too, if I understand the phrase correctly.
',' could be placed before 'so that', but it is not critical.

Reply:
You meant the coma was reduntant)) I thought the whole phrase! Yes, you're right!